eventHandlers.js

```diff
@@ -64,9 +63,8 @@ execute.loadingevents = function () {
     (event.currentTarget).classList.toggle("card");  
```

Comment:
This toggle is upsetting the flow of your app. When I click on a card not only should the input reset, all cards except the one clicked on should reset as well. With toggle you have to do it manually.